port_ocean/cli/cookiecutter/extensions.py

```diff
@@ -0,0 +1,12 @@
+from jinja2 import nodes
+from jinja2.ext import Extension
+from jinja2.parser import Parser
+
+from port_ocean import __version__
+
+
+class VersionExtension(Extension):
+    tags = {"version"}
+
+    def parse(self, parser: Parser) -> nodes.Node | list[nodes.Node]:
+        return nodes.Const(__version__, lineno=next(parser.stream).lineno)
```

Comment:
I am less familiar with how this fits in the general flow, I would suggest throwing a comment there with why & how